config/application.rb

```diff
@@ -21,23 +21,30 @@ class Application < Rails::Application
     config.i18n.default_locale = :en
     available_locales = [
       "ar",
+      "bg",
       "bs",
+      "ca",
       "cs",
       "da",
       "de",
       "el",
       "en",
       "es",
+      "es-PE",
```

Comment:
Yes, I thought the same thing. The only reason I enabled it is because the fallback language (es) is complete for this special case. I applied the same reasoning for Euskara.

Reply:
:ok_hand:.